source/documentation/deploying_services/postgresql.md.erb

```diff
@@ -766,48 +766,47 @@ cf create-service postgres large-11 my-pg-service-copy  -c '{"restore_from_point
 will create a new database from the write-ahead logs before October 27th 2020
 13:00 UTC.
 
-## Upgrading from Postgres 9.5
+## Upgrading major versions of Postgres
 
-Unfortunately, there is no straight path from RDS PostgreSQL 9.5 to RDS PostgreSQL 12. If you attempt to perform such an upgrade, you will get the following error message:
+You can upgrade from one major version to the next by performing a plan upgrade.
 
-```
-$cf update-service accounts-db -p tiny-unencrypted-12
-Updating service instance accounts-db as <REDACTED>@digital.cabinet-office.gov.uk...
-FAILED
-Server error, status code: 502, error code: 10001, message: Service broker error: InvalidParameterCombination: RDS does not support creating a DB instance with the following combination: DBInstanceClass=db.t3.micro, Engine=postgres, EngineVersion=9.5.23, LicenseModel=postgresql-license. For supported combinations of instance class and database engine version, see the documentation.
-$
-```
+There are two constraints on major version changes:
+
+1. You cannot downgrade
+1. You can only upgrade 1 major version at a time. E.g. 10 to 11, not 10 to 12.
 
-#### Upgrade to PostgreSQL 10
-To upgrade from Postgres 9.5, first you should upgrade your PostgreSQL 9.5 database to PostgreSQL 10.
+
+To perform the plan upgrade, run the following code at the command line:
 
 ```
-$ cf update-service accounts-db -p tiny-unencrypted-10
+$cf update-service SERVICE_NAME -p PLAN_NAME
```

Comment:
OOI why no space between `$` and `cf`?

Reply:
Oh, I see a lot of this was already like that

Reply:
Yep. I wasn't sure of the style, so I copied the rest of the document.